pkg/kudoctl/cmd/bundle.go

```diff
@@ -0,0 +1,74 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/kudobuilder/kudo/pkg/kudoctl/bundle"
+	"github.com/spf13/cobra"
+)
+
+const (
+	example = `
+		The bundle argument must be a  directory which contains the operator definition files.  The bundle command will create are tgz file containing the operator.
+
+		# Bundle zookeeper (where zookeeper is a folder in the current directory)
+		kubectl kudo bundle zookeeper
+
+		# Specify an destination folder other than current working directory
+		kubectl kudo bundle ../operators/repository/zookeeper/operator/ --destination=<target_folder>`
```

Comment:
since this is an example, maybe we can just make up a target folder instead of using placeholder?